src/hand_score.rs

```diff
@@ -158,63 +148,63 @@ mod tests {
         assert_eq!(
             card_names_to_hand_score(&deck, &["5♣", "8♣", "8♠", "7♣", "9♦"]),
             HandScore {
-                pair: true,
-                high_card: true,
+                pair: true as u32,
+                high_card: true as u32,
                 ..Default::default()
             }
         );
 
         assert_eq!(
             card_names_to_hand_score(&deck, &["5♣", "9♣", "8♣", "7♣", "2♣"]),
             HandScore {
-                flush: true,
-                high_card: true,
+                flush: true as u32,
+                high_card: true as u32,
                 ..Default::default()
             }
         );
         assert_eq!(
             card_names_to_hand_score(&deck, &["5♣", "4♦", "7♣", "7♦", "4♥"]),
             HandScore {
-                pair: true,
-                two_pair: true,
-                high_card: true,
+                pair: true as u32,
+                two_pair: true as u32,
+                high_card: true as u32,
                 ..Default::default()
             },
         );
         assert_eq!(
             card_names_to_hand_score(&deck, &["5♣", "4♦", "7♣", "5♦", "5♥"]),
             HandScore {
-                pair: true,
-                three_of_a_kind: true,
-                high_card: true,
+                pair: true as u32,
+                three_of_a_kind: true as u32,
+                high_card: true as u32,
```

Comment:
Here and below it would be fine to just use `0` and `1`, as we're checking a total count here, rather than doing Boolean logic. But fine to leave as is.

Reply:
I want to solve all the issues it's just that I get a little time to spend these days. I will keep contributing as much as I can and whenever I get time

Reply:
That sounds good!  I'm also a bit short on time these days.